models/metrics/metrics.yml

```diff
@@ -0,0 +1,181 @@
+version: 2
+sources: []
+models: []
+metrics:
+- name: segment__daily_number_of_sessions
+  label: Segment Daily Number of Sessions
+  model: ref('segment_web_sessions')
+  description: Number of sessions daily
+  type: count_distinct
+  sql: session_id
+  timestamp: session_start_tstamp
+  time_grains: [day]
+  filters: []
+  dimensions: []
+
+- name: segment__daily_number_of_sessions_by_device_category
+  label: Segment Daily Number of Sessions By Device Category
+  model: ref('segment_web_sessions')
+  description: Number of sessions daily differentiated by device category
+  type: count_distinct
+  sql: session_id
+  timestamp: session_start_tstamp
+  time_grains: [day]
+  filters: []
+  dimensions: [device_category]
+
+- name: segment__daily_number_of_sessions_by_referrer_source
+  label: Segment Daily Number of Sessions By Referrer Source
+  model: ref('segment_web_sessions')
+  description: Number of sessions daily differentiated by where they are coming from
```

Comment:
Let's make this `Daily number of sessions segmented by the site that sent the traffic`